core/src/main/java/org/mule/runtime/core/api/artifact/ArtifactCoordinates.java

```diff
@@ -11,20 +11,6 @@
  *
  * @since 4.5.0
```

Comment:
add `@deprecated` with explanation in the javadoc as well

Reply:
Does it make sense to deprecate something that was added "since 4.5"?